### PR TITLE
Update IDE.md

### PR DESCRIPTION
**Pull request by vineetgupta1991**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [x] Have you added copyright headers to new files?
* [x] Have you checked that both Scala and Java APIs are updated?
* [x] Have you updated the documentation for both Scala and Java sections?
* [x] Have you added tests for any changed functionality?

# Helpful things

## Fixes

Fixes #xxxx

## Purpose

What does this PR do?

## Background Context

Why did you take this approach?

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @vineetgupta1991,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by vineetgupta1991** [comment]
Added more information before running the command to eclipsify the project

File: documentation/manual/gettingStarted/IDE.md [hunk 1]
```diff
@@ -9,7 +9,13 @@ However, using a modern Java or Scala IDE provides cool productivity features li
 
 ### Generate configuration
 
-Play provides a command to simplify Eclipse configuration. To transform a Play application into a working Eclipse project, use the `eclipse` command:
+Play provides a command to simplify Eclipse configuration. Before running the command, you need to add below plugin in your plugin.sbt:
+
+```
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
```
Comment (PromanSEW) [inline documentation/manual/gettingStarted/IDE.md]: Latest version is 5.2.4

**Comment by vineetgupta1991** [comment]
Hi @marcospereira - Yes, basically I was facing the problem while eclipsifying my project in Play 2.3. After adding that plugin, it works fine. Play 2.6 looks okay for me but our current project works on 2.3.

**Pull request closed without merging**